File: service/src/main/java/hu/bsstudio/robonaut/race/speed/timer/DefaultSpeedTimerService.java
package hu.bsstudio.robonaut.race.speed.timer;

import hu.bsstudio.robonaut.race.speed.timer.model.SpeedTimer;
import lombok.RequiredArgsConstructor;
import reactor.core.publisher.Mono;

@RequiredArgsConstructor
public class DefaultSpeedTimerService implements SpeedTimerService {

    @Override
    public Mono<SpeedTimer> updateTimer(final SpeedTimer speedTimer) {
        return Mono.just(speedTimer);
    }
}

File: server/service/src/main/kotlin/hu/bsstudio/robonaut/race/speed/timer/DefaultSpeedTimerService.kt
package hu.bsstudio.robonaut.race.speed.timer

import hu.bsstudio.robonaut.race.speed.timer.model.SpeedTimer
import reactor.core.publisher.Mono

class DefaultSpeedTimerService : SpeedTimerService {
    override fun updateTimer(speedTimer: SpeedTimer): Mono<SpeedTimer> {
        return Mono.just(speedTimer)
    }
}
